kong/db/strategies/connector.lua

```diff
@@ -5,8 +5,8 @@ local fmt = string.format
 local Connector = {
```

Comment:
Should we do `if database == "off" then` here instead?

Reply:
Perhaps there is no kong.configuration at that time.

Reply:
It can not work, since `kong.configuration` is nil when the module is required.